content/blog/2022/10/2022-10-20-jenkins-election-announcement.adoc

```diff
@@ -0,0 +1,109 @@
+---
+:layout: post
+:title: "Jenkins Board and Officer Elections 2022"
+:tags:
+- community
+- governance
+- governance-board
+- elections
+authors:
+- kmartens27
+description: >
+    Nomination and voter registration period 2022 now open for Jenkins Board elections.
+links:
+  discourse: true
+opengraph:
+  image: /images/governance/elections/2022/2022-opengraph.png
+---
+
+We are excited to announce the 2022 Jenkins Governance Board elections!
+The nomination and voter registration period is now open.
+Nominations can be submitted for two link:/project/board[governance board] positions, and all five link:/project/team-leads/[officer] positions (Security, Events, Release, Infrastructure, and Documentation).
+
+Many thanks to link:https://github.com/halkeye[Gavin Mogan] and link:https://github.com/ewelinawilkosz[Ewelina Wilkosz] for serving on the link:https://www.jenkins.io/project/board/[Jenkins Governance Board].
+We also want to thank link:https://github.com/dduportal[Damien Duportal] as link:https://www.jenkins.io/project/team-leads/#infrastructure[Infrastructure Lead], link:https://github.com/wadeck[Wadeck Follonier] as link:https://www.jenkins.io/project/team-leads/#security[Security Officer], link:https://github.com/alyssat[Alyssa Tong] as link:https://www.jenkins.io/project/team-leads/#events[Events Officer], link:https://github.com/timja[Tim Jacomb] as link:https://www.jenkins.io/project/team-leads/#release[Release Officer], and link:https://github.com/MarkEWaite[Mark Waite] as link:https://www.jenkins.io/project/team-leads/#documentation[Documentation Officer] for all of their work over the last term. 
```

Comment:
should use relative links for all www.jenkins.io links so that preview site / development doesn't send you off to production site

Reply:
Ah, that definitely makes sense. I was able to update the links to be relative, and still work as intended. Thanks very much @timja!